Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Miden-node Dockerfile
+
+#### Setup image builder
+FROM rust:1.75.0-bullseye AS builder
```

Comment:
questions:

- why bullseye and not bookworm? (bookworm is the latest release)
- have you tried the slim version instead? (for smaller image sizes, even some other base image, like [alpine](https://hub.docker.com/_/alpine))

Reply:
Bullseye seemed the most used, how can I know which one is the latest one?

Reply:
The images are [listed here](https://hub.docker.com/_/debian),